### PR TITLE
Fix Vulnerability

### PR DESCRIPTION
**Pull request by bhubam**
## Description
Updated apache kafka version to remove vuln.

### CODE REVIEW

File: event-store/build.gradle.kts
Comment (aaron-steinfeld) [inline event-store/build.gradle.kts]: please use the kafka-bom here. Waiting for a review on https://github.com/hypertrace/kafka-streams-framework/pull/63

Also I'm guessing this repo needs to update to use the owasp dependency check, example can be found here: https://github.com/hypertrace/document-store/pull/158

Comment (aaron-steinfeld) [reply event-store/build.gradle.kts]: So far so good, next part would be to update the CI run so it runs the new scan instead of the old snyk scan

File: .github/workflows/pr-build.yml
Comment (aaron-steinfeld) [inline .github/workflows/pr-build.yml]: There's no docker image in this repo, it's just a lib. Can remove trivy. Instead, add the dependency scan job.

File: .github/workflows/pr-build.yml
Comment (aaron-steinfeld) [inline .github/workflows/pr-build.yml]: See example - it should use shared action and be a separate job

https://github.com/hypertrace/attribute-service/blob/main/.github/workflows/build-and-test.yml#L88

Comment (bhubam) [reply .github/workflows/pr-build.yml]: fixed

File: .github/workflows/pr-build.yml
Comment (aaron-steinfeld) [inline .github/workflows/pr-build.yml]: So this is the correct declaration of the job, but if you look at your checks it hasn't run. That's because this is on the pull_request_target workflow - which doesn't take effect until after merge. There's no reason to use that here, so we can either move this into the test workflow (which uses the pull_request trigger, will pick up the change in PR), or what would be even better is removing the `pull_request_target` workflow and scheduling the run anyway, since that's one of the to-do items for all these repos.

An example of that https://github.com/hypertrace/service-framework/pull/70

File: owasp.suppressions.xml
Comment (aaron-steinfeld) [inline owasp.suppressions.xml:1]: Check is working now, but failing due to a config issue (this file is misnamed):
```

Unable to read suppression file 'owasp-suppressions.xml'
Exception occurred initializing CPE Analyzer.
```

Comment (bhubam) [reply owasp.suppressions.xml:1]: my bad, fixed

**Review by aaron-steinfeld — APPROVED** [review]

**Review by aaron-steinfeld — APPROVED** [review]

**Review by avinashkolluru — APPROVED** [review]

**Pull request merged**
